addons/mail/static/tests/discuss_app/discuss.test.js

```diff
@@ -2230,23 +2231,23 @@ test("composer state: attachments save and restore", async () => {
 });
 
 test("sidebar: cannot leave channel with group_ids", async () => {
-    mockDate("2023-01-03 12:00:00"); // so that it's after last interest (mock server is in 2019 by default!)
```

Comment:
Note: seems irrelevant, test was badly written and didn't assert anything either way